components/alert/sidebar.css

```diff
@@ -0,0 +1,33 @@
+.aside-left{
+    width: 30%;
+    height: 100vh;
+   
+    /* padding: 20px; */
+    /* background-color: rgb(221, 151, 21);   */
+}
+
+h3{
+    font-size: 35px;
```

Comment:
Can this be refactored to rem ?

Reply:
yes.. of-course

Reply:
made the changes . please review

Reply:
removed the commented and unused code. please review .